spec/features/users_can_view_projects_spec.rb

```diff
@@ -47,7 +47,10 @@
   let(:urn) { 19283746 }
   let(:establishment) { build(:academies_api_establishment) }
 
-  before { mock_successful_api_establishment_response(urn: urn, establishment: establishment) }
+  before {
```

Comment:
I think now this is multi-line, it's more conventional to update it to a `do...end`. Seems like a contentious issue though https://stackoverflow.com/questions/5587264/do-end-vs-curly-braces-for-blocks-in-ruby